### ISSUE TITLE
Does Relu not work? 

### ISSUE THREAD

**Issue by Johnmakuta** [MikhailKravets/NeuroFlow#7]
So I was playing around the with example script for training a basic neural network that uses a Tanh activation function but I saw in the git repo, the implementation of the relu in activators. I wanted to try out Relu on the problem but it threw this error:

```bash
error[E0432]: unresolved import `neuroflow::activators::Type::Relu`
 --> src\main.rs:3:5
  |
3 | use neuroflow::activators::Type::Relu;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `Relu` in `activators::Type`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `hello_cargo` (bin "hello_cargo") due to previous error
```

example I used: [here](https://docs.rs/neuroflow/0.1.3/neuroflow/)

**Comment by Poxxy** [comment]
The package that is used when running `cargo add` is pulling a version older than this repo. You can go to the definitions for `activators::Type` which shows 

```
//! Module contains popular neural networks activation functions
//! and theirs derivatives

/// Determine types of activation functions contained in this module.
#[allow(dead_code)]
#[derive(Serialize, Deserialize)]
pub enum Type {
    Sigmoid,
    Tanh
}


pub fn sigm(x: f64) -> f64{ 1.0/(1.0 + x.exp()) }
pub fn der_sigm(x: f64) -> f64{
    sigm(x)*(1.0 - sigm(x))
}

pub fn tanh(x: f64) -> f64{
    x.tanh()
}

pub fn der_tanh(x: f64) -> f64{
    1.0 - x.tanh().powi(2)
}
```

This project hasn't been updated in a long time, so I would not count on any updates to the package itself.

**Comment by MikhailKravets** [comment]
Should be fixed in #8. Please feel free to re-open the issue if the error remains
